### PR TITLE
azurize optimus 

### PR DESCRIPTION
**Pull request by nikellepetrillo**
### Description

_Give your PR a **concise** yet **descriptive** title._
_Please explain the changes you made here._
_Explain the **motivation** for making this change. What existing problem does the pull request solve?_
_Mention any issues fixed, addressed, or otherwise related to this pull request, including issue numbers or hard links for issues in other repos._
_You can delete these instructions once you have written your PR description._

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

**Comment by kayleemathews** [comment]
This will require some minor doc updates @nikellepetrillo

File: pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.changelog.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 1.3.1
```
Comment (kayleemathews) [inline pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.changelog.md:1]: We should update the multi-snSS2 Overview and Methods docs with the new pipeline version.

File: pipelines/skylab/multiome/Multiome.wdl [hunk 2]
```diff
@@ -6,10 +6,11 @@ import "../../../tasks/skylab/H5adUtils.wdl" as H5adUtils
 import "https://raw.githubusercontent.com/broadinstitute/CellBender/v0.3.0/wdl/cellbender_remove_background.wdl" as CellBender
 
 workflow Multiome {
-    String pipeline_version = "3.2.1"
+    String pipeline_version = "3.2.2"
 
     input {
         String input_id
+        String cloud_provider
```
Comment (kayleemathews) [inline pipelines/skylab/multiome/Multiome.wdl:13]: need to add to pipeline inputs table

File: pipelines/skylab/slideseq/SlideSeq.wdl [hunk 3]
```diff
@@ -39,6 +41,32 @@ workflow SlideSeq {
         Boolean count_exons = true
         File bead_locations
 
+        String cloud_provider
```
Comment (kayleemathews) [inline pipelines/skylab/slideseq/SlideSeq.wdl:44]: need to add to pipeline inputs

**Review by kayleemathews — COMMENTED** [review]
@nikellepetrillo I made some comments about places that documentation needs to be updated for this PR. I know it isn't going into develop yet, so it isn't urgent, but I'll try to get to these today or tomorrow.

**Comment by nikellepetrillo** [comment]
retest this please

**Review by kayleemathews — APPROVED** [review]
Docs are updated!

**Comment by nikellepetrillo** [comment]
Optimus plumbing is failing in CompareH5ADOptimus. After comparing the h5ad files, it appears the differences are from EmptyDrops (nondeterminism)

**Comment by nikellepetrillo** [comment]
retest this please

**Pull request merged**